tfhe/src/core_crypto/algorithms/generate_accumulator.rs

```diff
@@ -0,0 +1,58 @@
+use super::super::prelude::{
+    allocate_and_trivially_encrypt_new_glwe_ciphertext, CastFrom, CastInto, CiphertextModulus,
+    GlweCiphertextOwned, GlweSize, PlaintextList, PolynomialSize, UnsignedTorus,
+};
+
+// Helper function to generate an accumulator for a PBS
```

Comment:
This needs a docstring

- explain delta allows to scale the message in the MSBs
- explain what the message modulus is here
- explain there is redundancy via "boxes" to reduce the noise during a PBS operation
- explain the accumulator is rotated by half a "box" to manage the negacyclic nature of the PBS
- link to a PBS docstring to show usage